scripts/bagging_variables.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+# This script will be sourced by the bmux command and the variables will be
+# used by the bag command. It is imperative that each variable is in the bag_*
+# namespace and contains a space diliniated list of ROS topics. The only two
+# exceptions to this rule are the BAG_ALWAYS and BAG_DIR variables.
+
+
+# Define topics that should be in every bag
+export BAG_ALWAYS="/odom /absodom /clock"
```

Comment:
Nobody should ever be typing this manually and BAG_ALWAYS just seemed more clear to me.